met-api/migrations/versions/17bae2d586fb_add_generated_document_tables.py

```diff
@@ -0,0 +1,64 @@
+"""Add generated document models
+
+Revision ID: 17bae2d586fb
+Revises: 904c1ebca3e3
+Create Date: 2022-12-27 16:16:50.850377
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from datetime import datetime
+
+
+# revision identifiers, used by Alembic.
+revision = '17bae2d586fb'
+down_revision = '904c1ebca3e3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    document_type = op.create_table('generated_document_type',
```

Comment:
generated in generated_document_type is a tricky word for a table name ; it might not be very clear to the other peeps ; 
just document_type ?

Reply:
Will change to document_generation_template